File: Paprogrames/C++/Paprogrames_03_pametrai.cpp
#include <iostream>
using namespace std;

void add(int a) {
    a += 10;
}

void add(int &a) {
    a += 10;
}

void greet(string name = "student") {
    cout << "Hello, " + name << endl;
}

int main() {
    int number = 100;
	
	add(number);
	cout << number << endl;
	
	add(number);
	cout << number << endl;
	
	greet();
	
    return 0;
}

File: Paprogrames/Python/Paprogrames_03_pametrai.py
def modify(x):
    x += 10
    return x

def modify_list(my_list):
    my_list.append(10)
    
def greet(name="student"):
    print("Hello, " + name)    

number = 3
print(modify(number))
print(number)
my_list = [1, 5]
print(modify_list(my_list))
greet()